models/Tag.js

```diff
@@ -33,6 +33,9 @@ module.exports = function(sequelize, DataTypes) {
     age: { // Guessed age in weeks of animal
       type: DataTypes.INTEGER
     },
+    automatic: { //
+      type: DataTypes.BOOLEAN
```

Comment:
I think we should also include the default value and allowNull here so it's the same as it was defined in the migration.

Reply:
I wasn't sure if those properties were meant to be here as well so I left them out.

Reply:
Of course it makes sense to have it though (for when starting from scratch?). Fixed.